code-challenges.test.js

```diff
@@ -14,56 +14,154 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const numbersArray1 = [6, 7, 8, 9, 10]
-// Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
-// Expected output: [72, 81, 90, 99, 108]
+        // EXAMPLE FROM coding-concepts.js
+        // var multBy2 = [4, 5, 6, 7, 8].map(value => value * 2)
+        // console.log(multBy2)
 
+// const numbersArray1 = [6, 7, 8, 9, 10]
+// // Expected output: [18, 21, 24, 27, 30]
+// const numbersArray2 = [24, 27, 30, 33, 36]
+// // Expected output: [72, 81, 90, 99, 108]
 
 // b) Create the function that makes the test pass.
 
 
-
 // --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
-
 // a) Create a test with expect statements for each of the variables provided.
 
-const num1 = 15
+// const num1 = 15
 // Expected output: "15 is divisible by three"
-const num2 = 0
+
+// const num2 = 0
 // Expected output: "0 is divisible by three"
-const num3 = -7
+
+// const num3 = -7
 // Expected output: "-7 is not divisible by three"
 
+        // Use 'describe ()', 'it ()' and 'expect ()' to test the syntax with jest.
+        
+//         describe("divby3", () => {
+//         const num1 = 15
+//         // Expected output: "15 is divisible by three"
+//         const num2 = 0
+//         // Expected output: "0 is divisible by three"
+//         const num3 = -7
+//         // Expected output: "-7 is not divisible by three"
+        
+//         it("takes a number as an argument and decides if the number is divisble by three", () => {
+//         expect(divby3(num1)).toEqual("15 is divisible by three")
+//         expect(divby3(num2)).toEqual("0 is divisible by three")
+//         expect(divby3(num3)).toEqual("-7 is not divisible by three")
+//        })
+//      })          
+
+        // after running yarn jest received
+        // --> ReferenceError: divby3 is not defined
```

Comment:
Let's make sure we are not sending code for review that is commented out. The code won't work if it's commented out.